### DISCUSSION TITLE
Chrome autocomplete not working

### DISCUSSION THREAD

**Discussion by jzamora5** [bl00mber/react-phone-input-2#446] [Q&A]
Hi, I am using the mask component and I saw that it has a prop called autocompleteSearch, however even if I set it to true or one of the other values of the attribute such as "on" my browser will not show auto complete suggestions. I have been searching around, and it seems it is necessary that the input element has an id, however there is no prop to pass said value. Any workaround or solution for this?

Thank you!

**Comment by sw00d** [comment]
Any update on this?

**Comment by himalay123456** [comment]
Autocomplete is not working for the safari browser. Any solution to support autocomplete ???
![WhatsApp Image 2023-02-23 at 10 10 04 PM](https://user-images.githubusercontent.com/88788155/220972360-dc55f62b-d30c-4b1e-a2ba-dae169ad2fb9.jpeg)

**Comment by rayanalhamiri777-collab** [comment]
778 789 850
